iroha/src/wsv.rs

```diff
@@ -258,13 +259,17 @@ pub mod config {
         /// [`MetadataLimits`] of any account's metadata.
         #[serde(default = "default_account_limits")]
         pub account_metadata_limits: MetadataLimits,
+        /// [`LengthLimits`] of identifiers of items that can be stored in the WSV.
```

Comment:
Could you change this configuration and remove all `serde(default = "...")`.

`#[serde(default)]` can be added to whole structure as an attribute

Reply:
Done